main.js

```diff
@@ -21,4 +21,193 @@ closeIcon.addEventListener('click', disapear);
 
 links.forEach((link) => {
   link.addEventListener('click', disapear);
-});
\ No newline at end of file
+});
+
+// Popup Detail Window and Section 2 Content
```

Comment:
- [ ] [OPTIONAL] You can consider restricting users from scrolling the web page, because now, when I open the modal I still can scroll the page and the modal appears to be elsewhere 😆 
You can simply do it by adding `body{overflow: hidden` same time when modal has to appear, and remove it when users click `x`. This way users will not be able to scroll the site, and will only see modal.

- [ ] You can also consider adding blury dark color around the modal like in Figma 😉 

First Header  | Yours
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/146628855-c1636bb5-ca8d-4105-9470-c02ec128025a.png) | ![image](https://user-images.githubusercontent.com/77038610/146628815-62cc396f-e6c3-4485-96e9-1d5294984867.png)